lib/src/qr_code_scanner.dart

```diff
@@ -334,7 +334,7 @@ class QRViewController {
         await channel.invokeMethod('setDimensions', {
           'width': renderBox.size.width,
           'height': renderBox.size.height,
-          'scanArea': overlay?.cutOutSize ?? 0,
+          'scanArea': overlay?.cutOutWidth ?? 0, //Todo
```

Comment:
What is this todo comment?

Reply:
This is just for me)
I will finish the feature soon and remove the todo comment

Reply:
This is just for me)
In order not to forget to complete the use of the real scanning area on ios
I will finish the feature soon and remove the todo comment

Reply:
I did it